app/src/components/keycloakAdminService.js

```diff
@@ -66,10 +69,11 @@ class KeycloakAdminService {
       baseUrl: this._baseUrl,
       realmName: this._realm
     });
-    this.initialize();
+    this.initialize(true);
   }
 
-  async initialize() {
+  async initialize(force = false) {
+    if (force === true) this._initialized = false;
```

Comment:
Can depend on truthiness instead of needing to check explicitly.
`if (force) this._initialized = false;`

Reply:
That's not really true.  There is a difference between checking if the parameter is something or checking if the parameter is in fact true. initialize(true) and initialize('force') are 2 different things.